### ISSUE TITLE
Insights is connected to the DB but it not loading the UI

### ISSUE THREAD

**Issue by kitunzit** [frappe/insights#288]
Good day.
Insight app has connected to the MariaDB, on the second step/window showing " Help Us improve" i press skip & two messages are displayed  " internal server error" & "Syncing Data source. This may take a while. Please wait....". See attached image.
Am running ERPNext 15 hosted on cloud clusters.
Any help will be appreciated. 
Thanks 
![Image 31-07-2024 at 12 07 PM](https://github.com/user-attachments/assets/39dc0ec8-b694-4be2-8491-b4793c04a13d)

**Comment by nextchamp-saqib** [comment]
Please share the traceback from the browser console. To do that open the browser console after you get "Internal Sever Error" and send the last error message

**Comment by sunil-mel** [comment]
I was also getting same error with postgresdb. Attached screenshot of the same. I modified the code in the file `insights_data_source_v3.py` line 62 as below to make it work.

```
self.db_set("is_frappe_db", 0)
```

![image](https://github.com/user-attachments/assets/747a94bd-b5d6-4cc9-8612-3d3d9744b223)

Next it was failing in the file `ibis_utils.py` line 388. I replaced with below code to make it work.

```
res = res.replace({pd.NaT: None, np.nan: None})
```
